Documentation/DataSources/Limits.md

```diff
@@ -0,0 +1,53 @@
+# Limits
+
+
+## Data Source Limits
+
+### General
+* In general, Workbooks limits the results of queries to be no more than 10,000 results. Any results after that point are truncated.
+* Each data source may have its own specific limits based on the limits of the service they query.
+* Those limits may be on the numbers of resources, regions, results returned, time ranges.  Consult the documentation for each service to find those limits.
+
+### Log based Queries (Log Analytics, Application Insights, resource centric queries)
+* Log Analytics [has limits](https://docs.microsoft.com/en-us/azure/azure-monitor/service-limits#log-queries-and-language) for the number of resources, workspaces, and regions involved in queries.
```

Comment:
don't we have a 250 limit on the query?

Reply:
no, the grid limits rows that it shows, by default.  the hard query limit is 10k